src/main/resources/mapper/StudentMapper.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN"
+        "http://mybatis.org/dtd/mybatis-3-mapper.dtd" >
+<mapper namespace="com.demo.MybatisApplication.repository.StudentRepository">
+
+    <resultMap id="StudentResultMap" type="com.demo.MybatisApplication.model.StudentEntity">
+        <id property="id" column="id"/>
+        <result property="studentName" column="student_name"/>
```

Comment:
```suggestion
        <result property="studentName" column="name"/>
```
Follow the above guideline while naming the columns

Reply:
Done